modules/nw-ovn-k-anp-best-practices.adoc

```diff
@@ -0,0 +1,36 @@
+/ Module included in the following assemblies:
+//
+// * list of assemblies where this module is included:
+// networking/network_security/AdminNetworkPolicy/nw-ovn-k-anp-recommended-practices.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="anp-best-practices_{context}"]
+= Designing AdminNetworkPolicy
+
+When building `AdminNetworkPolicy` (ANP), you should consider the following when creating your policies:
+
+//This will need to be added in at 4.16.1
+
+//* You should create `AdminNetworkPolicy` at different priorities so that precedence is well defined. It is not guaranteed which policy will take precedence when overlapping ANP are created.
+
+* There is only one `BaselineAdminNetworkPolicy` for the cluster so be careful in defining it and you might consider using it as a default guardrail policy.
+
+* Since `0-100` is the supported priority range, it is always better when designing your ANP to use a middle range like `30-70` because this leaves some placeholder
+for priorities before and after. Even in the middle range, you might want to leave gaps so that as your infrastructure requirements evolve over time, you are able to insert new ANPs when needed at the right priority level. If you pack your ANPs, then you might need to recreate all of them to accommodate any changes in the future.
+
+* `networks` peer in egress rule is meant to select external destinations. It is recommended to use `namespaces` and `pods` peers to select in-cluster peers. Do not use `networks` for podIP level matching.
```

Comment:
I don't think this is a useful/correct recommendation. The reason `networks` is not recommended for pod IP matching is because it's not _useful_ for that, and this should be completely obvious without us needing to point it out. But if the admin has some strange use case where for some reason using `networks` works better than `namespaces` or `pods`, then they should do that, and it is supported.

Reply:
So just remove this?